docs/operating-eck/eck-permissions.asciidoc

```diff
@@ -0,0 +1,112 @@
+:page_id: eck-permissions
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+
+[id="{p}-{page_id}"]
+= ECK permissions
+
+Installing and running ECK, as well as using ECK-managed resources requires certain Kubernetes link:https://kubernetes.io/docs/reference/access-authn-authz/rbac[permissions]. Each of those permissions is listed below, together with a reason why it is needed.
+
+
+[float]
+[id="{p}-{page_id}-defaults"]
+== Installing CRDs
+
+Below is the permission required to install CRDs. CRDs (link:https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/[CustomResourceDefinitions]) are the only non-namespaced resources required to be installed.
+
+[width="100%",cols=".^35m,.^25m,.^5d,.^35",options="header"]
+|===
+|Name|API group|Optional?|Usage
+|CustomResourceDefinition|apiextensions.k8s.io|no|Extend Kubernetes APIs with Elastic Stack applications resources
+|===
+
+[float]
+[id="{p}-{page_id}-installing"]
+== Installing the ECK operator
+
+Below are the permissions required to install ECK operator in a Kubernetes cluster.
+
+[width="100%",cols=".^1m,.^1m,.^1d,.^97",options="header"]
+|===
+|Name|API group|Optional?|Usage
+|StatefulSet or Deployment|apps|no|ECK operator `Pod` deployment method. By default, `Deployment` is used only in OpenShift.
```

Comment:
deployment is a bit overloaded here. Also the  `Deployment`  is not restricted to OCP but all installations that are using OLM. Maybe simply: "The ECK operator can be either deployed as a StatefulSet or as a Deployment"